src/CmdLineParser.cc

```diff
@@ -266,6 +279,15 @@ void CmdLineParser::SetPreprocessor(const char *arg)
     m_Preprocessor = p;
 }
 
+void CmdLineParser::SetTranslator(const char *arg)
+{
+    CodepageTranslator *t = new CodepageTranslator();
```

Comment:
This pointer leaks. Or at least I don't see it being deleted anywhere. But I can take care of this in master.

Reply:
No, i did not implement any cleanup code for the pointers cause i think there is no case where a translator is replaced during interpretation for several times and the program will end when one file was converted. So the operating system will take care of that cleanup.
Do you want me to implement some cleanup of the pointer?

Reply:
Of course no memory leak will matter after program exit, but I think it's bad practice. If for no other reason, then e.g. valgrind will pester you about it. But you can leave this as is, I plan to bring up the source code to a more modern C++, and I'll consider a smart pointer for this.

Reply:
Ok, so we do not touch it with this mr but maybe i will open a new one which can fix this (And for the preprocessor as well

Reply:
OK.

(By the way, I think the preprocessor instance does not leak, because the returned pointer points to a global instance that is destroyed at shutfown. But it's not so obvious. At first (seeing the code after many years) I thought it does leak too.)